### PR TITLE
Add R installation capability

### PR DESCRIPTION
**Pull request by tnederlof**
This is a first attempt at building out the feature described in: https://github.com/dpastoor/wbi/issues/8

It is structured so first it detects versions of R.
If R is installed but not in a /opt location then it gives the user the option of installing 1 or more versions.
If R is not installed at all then it gives the user the option of installing 1 or more versions (if not then it ends the program as R is required)

The prompt allows a user to select from a list of versions (right now the same list as on our docs site, although its hardcoded and should be done dynamically at some point). Then it loops through each version, downloads the correct installer into a tmp directory, runs any prerequisites (upgrades apt and installs gdebi-core if Ubuntu, optional repo in RHEL) and then installs the R version.

Video below:

https://user-images.githubusercontent.com/180123/218505422-88927a21-683f-4085-97fc-cf0d7f725d03.mp4

### CODE REVIEW

File: internal/languages/installR.go [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+	"io"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/dpastoor/wbi/internal/system"
+)
+
+var availableRVersions = []string{
```
Comment (dpastoor) [inline internal/languages/installR.go:16]: is this something we can query instead of hard coding?

File: internal/languages/installR.go [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+		return "sudo yum install -y " + filepath, nil
+	default:
+		return "", errors.New("operating system not supported")
+	}
+}
+
+func PopulateInstallerInfo(rVersion string, osType string) (InstallerInfo, error) {
+	switch osType {
+	case Ubuntu18:
+		return InstallerInfo{
```
Comment (dpastoor) [inline internal/languages/installR.go:207]: lets talk about some nitpicks on this implementation in our next live code review

**Review by dpastoor — COMMENTED** [review]
wip - just dropping comment before we meet

File: internal/languages/installR.go (hunk 1 above)
Comment (tnederlof) [reply internal/languages/installR.go:16]: Yes! I started down that path and couldn't figure out where we pull from so figured better to stub it like this then investigate a better way to query it.

**Review by dpastoor — APPROVED** [review]
good to get in then start refactoring

**Pull request merged**